### ISSUE TITLE
Oddly formatted student names not being parsed

### ISSUE THREAD

**Issue by jhthenerd** [al-caughey/Google-Meet-Attendance#96]
I'm a student hoping to use your extension to take attendance at club meetings, and I noticed a bug in the regex parsing of some names.

Our board has an odd student naming scheme and the extension seems to filter out the back half of the student login.

For example, this is a valid name:
`JH - 12FS 736813 John Fraser SS`

That gets truncated to this with the regex filtering:
`JH -`

It seems like the issue is with [this line of regex](https://github.com/al-caughey/Google-Meet-Attendance/blob/de33a04a391497042eb2567a63823a92dacf8443/src/js/translations.js#L32), that somehow [matches the back half of the username](https://regexr.com/5gsqr).

I'd love to open a PR to help fix this, but unfortunately, I'm not sure what the regex is supposed to look for.

**Comment by Eagle-Man** [comment]
@al-caughey Make sure you look at this before you push out the update with the other RegEx fix. ;)

**Comment by al-caughey** [comment]
sorry for taking so long to get to this.  Are you in the Peel Board of Ed?

Yes, that regex is causing problems...  I'll have to think about what to do about this

**Comment by jhthenerd** [comment]
Yup, I'm part of the Peel board. The pattern that our board follows for naming seems to be: 

`{2 letter initials} - {2 digit grade}{optional 2 letter identifier} {Student number} {School name}`
